packages/vm/lib/evm/interpreter.ts

```diff
@@ -206,7 +210,7 @@ export default class Interpreter {
         i += code[i] - 0x5f
       }
 
-      if (curOpCode === 'JUMPDEST') {
+      if (curOpCode === 'JUMPDEST' || curOpCode === 'BEGINSUB') {
```

Comment:
This is wrong, it lets JUMPSUB to go to a JUMPDEST. The new opcodes need their own valid jump table.

EIP spec for JUMPSUB says:
> If the opcode at location is not a BEGINSUB abort

Reply:
Addressed by ba3a64e